quickwit/quickwit-actors/src/tests.rs

```diff
@@ -374,7 +378,7 @@ impl Handler<SingleShot> for LoopingActor {
     }
 }
 
-#[tokio::test]
+#[tokio::test(flavor = "multi_thread")]
 async fn test_looping() -> anyhow::Result<()> {
```

Comment:
I time-shifted the universe by `HEARTBEAT * 2`, hoping to reproduce the issue, but that did not work. Same when sleeping for the same amount of time.